src/main/groovy/com/bugsnag/android/gradle/BugsnagVariantOutputTask.groovy

```diff
@@ -30,62 +31,105 @@ class BugsnagVariantOutputTask extends DefaultTask {
      * See: https://developer.android.com/studio/build/configure-apk-splits.html#build-apks-filename
      * https://issuetracker.google.com/issues/37085185
      */
-    File getManifestPath() {
+    List<File> getManifestPaths() {
         File directory
-        def processManifest = BugsnagPlugin.resolveProcessManifest(variantOutput)
-        def outputDir = processManifest.manifestOutputDirectory
+        File directoryBundle
+        List<File> manifestPaths = new ArrayList<>()
 
-        if (outputDir instanceof File) {
-            directory = outputDir
-        } else {
-            // gradle 4.7 introduced a provider API for lazy evaluation of properties,
-            // AGP subsequently changed the API from File to Provider<File>
-            // see https://docs.gradle.org/4.7/userguide/lazy_configuration.html
-            directory = outputDir.get().asFile
+        boolean getAssembleManifest = BugsnagPlugin.isRunningAssembleTask(variant, variantOutput, project)
+        boolean getBundleManifest = BugsnagPlugin.isRunningBundleTask(variant, variantOutput, project)
+
+        // If the manifest location could not be reliably determined, attempt to get both
+        if (!getAssembleManifest && !getBundleManifest) {
+            getAssembleManifest = true
+            getBundleManifest = true
         }
 
-        File manifestFile = Paths.get(directory.toString(), variantOutput.dirName, "AndroidManifest.xml").toFile()
+        def processManifest = BugsnagPlugin.resolveProcessManifest(variantOutput)
 
-        if (!manifestFile.exists()) {
-            project.logger.error("Failed to find manifest at ${manifestFile}")
-        } else {
-            project.logger.info("Found manifest at ${manifestFile}")
+        if (getAssembleManifest) {
+            def outputDir = processManifest.manifestOutputDirectory
+
+            if (outputDir instanceof File) {
+                directory = outputDir
+            } else {
+                // gradle 4.7 introduced a provider API for lazy evaluation of properties,
+                // AGP subsequently changed the API from File to Provider<File>
+                // see https://docs.gradle.org/4.7/userguide/lazy_configuration.html
+                directory = outputDir.get().asFile
+            }
+
+            File manifestFile = Paths.get(directory.toString(), variantOutput.dirName, "AndroidManifest.xml").toFile()
+
+            if (!manifestFile.exists()) {
+                project.logger.error("Failed to find manifest at ${manifestFile}")
+            } else {
+                project.logger.info("Found manifest at ${manifestFile}")
+                manifestPaths.add(manifestFile)
+            }
+        }
+
+        // Attempt to get the bundle manifest directory if required
+        if (getBundleManifest) {
+            if (!processManifest.hasProperty("bundleManifestOutputDirectory")) {
```

Comment:
Is it not possible to access this value at all with AGP 3.2.1? Here's an [example](https://github.com/bugsnag/bugsnag-android-gradle-plugin/blob/master/src/main/groovy/com/bugsnag/android/gradle/BugsnagPlugin.groovy#L246) of how we conditionally access a property elsewhere that is only available in later versions of the plugin.

Reply:
The property itself isn't available until AGP 3.3.0. I'll see if we can determine the bundle manifest directory a different way if the property is not available, perhaps it is at a consistent relative path to the merged manifest directory,

Reply:
Actually, the property doesn't exist for AGP 3.2.1 because the bundle manifest is the same as the merged manifest. The bundle manifest got its own directory for 3.3.0. That means that the issue that this PR is trying to solve only presents itself for AGP versions >= 3.3.0. The build UUID gets added to the bundle manifest ok for AGP 3.2.1.